docs/contributing/documentation_guidelines.md

```diff
@@ -89,7 +89,7 @@ Use **bold** and _italic_ sparingly and inline `code` when relevant.
 Use **bold** and _italic_ sparingly and inline `code` when relevant.
 ```
 
-Keep in mind that in reStructuredText, italic is written with `*`, and inline code must be written with double backticks, like ``` ``code`` ```.
+Keep in mind that in reStructuredText, italic is written with `*`, and inline code must be written with double backticks, like ` ``code`` `.
```

Comment:
Rendered like this, no need for the additional backticks to show the backticks as code.

<img width="882" alt="Screenshot 2024-12-04 at 8 17 23 AM" src="https://github.com/user-attachments/assets/96630123-2c54-41e1-a2f4-87912a5ece08">